fastlane/lib/fastlane/actions/testfairy.rb

```diff
@@ -9,9 +9,10 @@ def self.upload_build(ipa, options)
         require 'faraday'
         require 'faraday_middleware'
 
-        connection = Faraday.new(url: "https://app.testfairy.com") do |builder|
+        connection = Faraday.new(url: "https://upload.testfairy.com") do |builder|
           builder.request :multipart
           builder.request :url_encoded
+          builder.request :retry, max: 3, interval: 5
```

Comment:
TestFairy always overwrites builds with same build number, so operation should be safe.